src/Dapr.Client/DaprApiException.cs

```diff
@@ -12,79 +12,79 @@ namespace Dapr.Actors
     /// Exception for Dapr operations.
     /// </summary>
     [Serializable]
-    public class DaprException : Exception
+    public class DaprApiException : DaprException
     {        
         /// <summary>
-        /// Initializes a new instance of the <see cref="DaprException"/> class with error code 'UNKNOWN'"/>.
+        /// Initializes a new instance of the <see cref="DaprApiException"/> class with error code 'UNKNOWN'"/>.
         /// </summary>
-        public DaprException()
-            : this(Constants.Unknown, false)
+        public DaprApiException()
+            : this(errorCode: null, false)
         {
         }
 
         /// <summary>
-        /// Initializes a new instance of the <see cref="DaprException"/> class with error code 'UNKNOWN' and a specified error message.
+        /// Initializes a new instance of the <see cref="DaprApiException"/> class with error code 'UNKNOWN' and a specified error message.
         /// </summary>
         /// <param name="message">The error message that explains the reason for the exception.</param>
-        public DaprException(string message)
-            : this(message, Constants.Unknown, false)
+        public DaprApiException(string message)
+            : this(message, errorCode: null, false)
         {
         }
 
         /// <summary>
-        /// Initializes a new instance of the <see cref="DaprException"/> class with a specified error
+        /// Initializes a new instance of the <see cref="DaprApiException"/> class with a specified error
         /// message and a reference to the inner exception that is the cause of this exception.
         /// </summary>
         /// <param name="message">The error message that explains the reason for the exception.</param>
         /// <param name="innerException">The exception that is the cause of the current exception.</param>
-        public DaprException(string message, Exception innerException)
-            : this(message, innerException, Constants.Unknown, false)
+        public DaprApiException(string message, Exception innerException)
```

Comment:
Is DaprApiException like DaprActorException...the naming isn't very clear

Reply:
What's your suggestion? I kept the structure of these two types based on the existing places they are thrown and handled in Actors.

Reply:
DaprActorException

Reply:
Discussed offline.